Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -94,7 +78,7 @@ public int Buy(string symbol, double quantity)
         /// </summary>
         /// <param name="symbol">string Symbol of the asset to trade</param>
         /// <param name="quantity">decimal Quantity of the asset to trade</param>
-        /// <seealso cref="Order(string, double)"/>
+        /// <seealso cref="Buy(string, double)"/>
```

Comment:
Why was this changed? The seealso to Order() is on purpose, if anything, it should be updated to `<seealso cref="MarketOrder(string, int)">`